compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -1074,6 +1074,7 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
       s"""object $className extends $companionType {
          |  implicit def messageCompanion: $companionType = this""")
       .indent
+      .add(s"type ValueType = $className")
```

Comment:
Is it not the same thing? We get EnumType = ClassName and the being able to reference this type elsehwere. Correct me if I'm wrong.